packages/astro/types.d.ts

```diff
@@ -1,6 +1,7 @@
 import './astro-jsx';
 import type { OmitIndexSignature, Simplify } from './dist/type-utils.js';
 import type { AstroBuiltinAttributes } from './dist/types/public/elements.js';
+import type { BundledLanguage, SpecialLanguage, LanguageRegistration } from 'shiki';
```

Comment:
I think we need to move this to a file in `src/types/public` instead, I need to investigate